### PR TITLE
operator [CI] horreum-operator (0.2.0)

### PR DESCRIPTION
**Pull request by rvansa**
### Updates to existing Operators

* [x] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [x] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [x] Have you tested an update to your Operator when deployed via OLM?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [x] Modify more than one operator
* [x] Modify an Operator you don't own
* [x] Rename an operator - please remove and add with a different name instead
* [x] Modify any files outside the above mentioned folders
* [x] Contain more than one commit. **Please squash your commits.**
* [ ]

### CODE REVIEW

File: operators/horreum-operator/0.2.0/horreum-operator.v0.2.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    name: Ståle W. Pedersen
+  - email: wreicher@redhat.com
+    name: Will Reichert
+  maturity: alpha
+  minKubeVersion: 1.16.0
+  provider:
+    name: Red Hat, Inc.
+    url: https://horreum.hyperfoil.io
+  version: 0.2.0
+  replaces: 0.1.0
```
Comment (J0zi) [inline operators/horreum-operator/0.2.0/horreum-operator.v0.2.0.clusterserviceversion.yaml:288]: @rvansa  please use `replaces: horreum-operator.v0.1.0` here.

**Comment by mvalarh** [comment]
@rvansa please fix: https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/1807#issuecomment-1303177486

**Comment by J0zi** [comment]
@rvansa any update?

**Comment by rvansa** [comment]
Hi @J0zi , sorry for the delay, this week we have a F2F and there's little time for any 'real' work. I hope to get this fixed early next week. Thanks for the patience.

**Comment by J0zi** [comment]
@rvansa any update on this?

**Comment by rvansa** [comment]
Closing temporarily, I'll reopen when I find some time to work on this.

**Pull request merged**